app/routes/admin/permissions.js

```diff
@@ -5,5 +5,8 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('Permissions');
+  },
+  beforeModel() {
```

Comment:
Create a `admin/permissions/index` route and put this `beforeModel` there instead. Then this will work fine